### PR TITLE
updates to code of conduct (version 1.1)

### PR DESCRIPTION
**Pull request by evykassirer**
from https://git.uwaterloo.ca/starcon/exec-eyes-only/issues/148

this is a first draft - very open to discussion about to most efficiently and effective word this stuff!

3 main TODOs:
- [x] secure way for people to report
- [x] pick second incident responder 
- [x] add date of merging when we merge for the version

### CODE REVIEW

File: _pages/coc.md [hunk 1]
```diff
@@ -74,7 +80,14 @@ communications pertaining to community business.
 
 ## Contact Us ##
 
-We won't share your contact information or the contents of your message with anyone other than conference executives.
+You have several options for contacting us:
+
+- send a private Slack message to Evy Kassirer (trained incident responder) or any other conference organizer
+- direct message [Evy](https://twitter.com/EvyKassirer) on Twitter
+- find a conference organizer (wearing TODO coloured shirts) or volunteer (wearing TODO coloured shirts) and talk to us in person
```
[the remark this answers is not in the document]
Comment (evykassirer) [reply _pages/coc.md]: yup, there's a few around the PR - will be figuring that stuff out as feedback happens on the general content

File: _pages/coc.md [hunk 2]
```diff
@@ -74,7 +80,14 @@ communications pertaining to community business.
 
 ## Contact Us ##
 
-We won't share your contact information or the contents of your message with anyone other than conference executives.
+You have several options for contacting us:
+
+- send a private Slack message to Evy Kassirer (trained incident responder) or any other conference organizer
+- direct message [Evy](https://twitter.com/EvyKassirer) on Twitter
```
Comment (annalorimer) [inline _pages/coc.md]: See my comment in https://git.uwaterloo.ca/starcon/exec-eyes-only/issues/148

Comment (evykassirer) [reply _pages/coc.md]: thanks! replied there

**Comment by evykassirer** [comment]
FYI 3 main TODOs:

- [ ] secure way for people to report
- [ ] pick second incident reporter 
- [ ] add date of merging when we merge for the version

File: _pages/coc.md [hunk 3]
```diff
@@ -74,7 +80,17 @@ communications pertaining to community business.
...
 ## Contact Us ##
 
-We won't share your contact information or the contents of your message with anyone other than conference executives.
+Note that Evy Kassirer and Arshia Mufti are StarCon's official incident responders, and Evy has participated in a Code of Conduct enforcement training with [Sage Sharp](http://sage.thesharps.us/). You are welcome to reach out to them or to any other conference organizers or volunteers.
+
+You have several options for contacting us:
+
+- Send a private Slack message to any [conference organizer](/#whos-running-this)
+- Direct message [Evy](https://twitter.com/EvyKassirer) or [Arshia](https://twitter.com/arshia__) on Twitter
+- Message Evy securely on [Keybase](https://keybase.io/evyk/)
```
Comment (evykassirer) [inline _pages/coc.md]: arshia do you want to also get keybase?

File: _pages/coc.md [hunk 4]
```diff
@@ -74,7 +80,17 @@ communications pertaining to community business.
 
 ## Contact Us ##
```
Comment (arshiamufti) [inline _pages/coc.md]: change this to "how to report a violation"?

File: _pages/coc.md (hunk 3 above)
Comment (arshiamufti) [reply _pages/coc.md]: here you go: https://keybase.io/arshiamufti

File: _pages/coc.md [hunk 5]
```diff
@@ -85,14 +101,34 @@ We won't share your contact information or the contents of your message with any
 <input type="submit" value="Send">
 </form>
 
+## Processing Code of Conduct reports
```
Comment (arshiamufti) [inline _pages/coc.md]: does this render properly? the text is italicized

File: _pages/coc.md (hunk 4 above)
Comment (evykassirer) [reply _pages/coc.md]: ehh it's linked to in a couple of other places and one is reporting a grievance which you would still use these things for but it wouldn't be for reporting a violation

are you ok keeping as is?

File: _pages/coc.md (hunk 5 above)
Comment (evykassirer) [reply _pages/coc.md]: yup, I think it's the html or something
screen shot 2017-12-31 at 7 46 27 pm

**Comment by evykassirer** [comment]
@arshiamufti if you're good with it as it is feel free to merge before you send eventbrite, otherwise feel free to make changes (but I won't be around until Jan 2)

**Pull request merged**